python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/utils.py

```diff
@@ -122,6 +122,9 @@ def get_complex_sample_connector_schema_config(
                                 "allowed": False,
                                 "reason_code": "SYSTEM_TABLE",
                             },
+                            "columns": {
```

Comment:
[1] Columns are required in the [schema config API docs](https://fivetran.com/docs/rest-api/api-reference/connector-schema/connector-schema-config). By updating the legacy test samples, the goal is that the legacy tests succeed with the updated `FivetranConnectorTableProps`.

Reply:
Historically this wasn't the case (https://github.com/dagster-io/dagster/pull/23631) and if the user did not customize which columns to sync for a table, the Fivetran API would not return column schema info - might be good to verify that this is not the case now and/or still make sure we have a fallback in case we don't get column level info

Reply:
That's good feedback - I reverted the test changes in [a171504](https://github.com/dagster-io/dagster/pull/25889/commits/a17150461a40a9bb9c733853de9046db8c6b8876) and made columns optional